tasks/install.yml

```diff
@@ -0,0 +1,68 @@
+---
+- import_tasks: apt_install.yml
+  when: ansible_pkg_mgr == "apt"
+
+- name: Add gitlab user
+  user:
+      name: git
+      shell: /bin/nologin
+      comment: 'GitLab'
+      state: present
+      groups: redis
+
+- name: Install bundler gem
+  gem:
+    name: bundler
+  become: yes
+  become_user: git
+
+- import_tasks: postgres_install.yml
+
+- name: Creates redis socket directory
```

Comment:
*Create

Reply:
And I doubt this will work when the socket is in /tmp because the directory is cleaned on reboot

Reply:
is this needed? if yes use systemd tmp files